client/src/lib/firestore/room.ts

```diff
@@ -51,7 +55,8 @@ export async function syncWebRing(
   linkUpdate: (roomName: string, update: RoomLinkInfo) => void,
 ) {
   let collectionRef = roomsCollection();
-  let docs = await getDocs(collectionRef);
+  let docs = await getDocs(query(collectionRef, where("hidden", "!=", true)));
```

Comment:
@bhaviksingh updated! 

Note that Firestore isn't able to run queries where a field _doesn't exist_, so I added `hidden: false` to all the rooms that should be visible. Probably worth that extra little bit of setup to avoid complicating the code.